docs/UserGuide.adoc

```diff
@@ -96,6 +96,26 @@ Returns Any person having names `Betsy`, `Tim`, or `John`.
 Deletes the specified person from the address book. Irreversible. +
 Format: `delete INDEX`
 
+****
+Updates the person at the specified `INDEX`.
```

Comment:
Overall good work! But you may consider if the operator only wants to update some certain fields but not all.

Reply:
Thanks for your suggestions. That can be considered as the phase 2 of the command which I can work it on.